ql/src/test/results/clientpositive/llap/list_bucket_dml_9.q.out

```diff
@@ -14,6 +14,16 @@ POSTHOOK: query: create table list_bucketing_static_part_n0 (key String, value S
 POSTHOOK: type: CREATETABLE
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@list_bucketing_static_part_n0
+SQL Query : "select "DB_ID" from "DBS""
+SQL Query : "select "DB_ID" from "DBS""
+SQL Query : "select "DB_ID" from "DBS""
+SQL Query : "select "DB_ID", "NAME", "DB_LOCATION_URI", "DESC", "OWNER_NAME", "OWNER_TYPE", "CTLG_NAME" , "CREATE_TIME", "DB_MANAGED_LOCATION_URI", "TYPE", "DATACONNECTOR_NAME", "REMOTE_DBNAME"FROM "DBS" where "NAME" = ? and "CTLG_NAME" = ?"
```

Comment:
Not sure if we need a space between "REMOTE_DBNAME" and FROM, but it does not related to the changes.

Reply:
I'm a little afraid that it will generate many `Query` logs to the original though.